templates/repo/branch/list.tmpl

```diff
@@ -57,6 +59,27 @@
                                             	<div class="bar bar-ahead" style="width: {{percentage .CommitsAhead .CommitsBehind .CommitsAhead}}%"></div>
 											</div>
 										</div>
+										{{end}}
+									</td>
+									<td class="right aligned">
+
+									{{if not .LatestPullRequest}}
+										{{if not .IsDeleted}}
+										<a href="{{$.RepoLink}}/compare/{{$.DefaultBranch | EscapePound}}...{{if ne $.Repository.Owner.Name $.Owner.Name}}{{$.Owner.Name}}:{{end}}{{.Name | EscapePound}}">
+                                        	<button id="new-pull-request" class="ui compact basic button">{{$.i18n.Tr "repo.pulls.compare_changes"}}</button>
+                                        </a>
+										{{end}}
+									{{else}}
+										#{{.LatestPullRequest.Issue.Index}}
+                                        {{if .LatestPullRequest.HasMerged}}
+                                        	<a href="{{$.RepoLink}}/pulls/{{.LatestPullRequest.Issue.Index}}" class="ui purple large label"><i class="octicon octicon-git-pull-request"></i> {{$.i18n.Tr "repo.pulls.merged"}}</a>
```

Comment:
Can you make these buttons a tiny bit smaller (less padding)? Maybe by just removing the `large` class.